modules/end-user-guide/partials/ref_parameters-for-che-editor-yaml.adoc

```diff
@@ -7,7 +7,7 @@ The simplest way to select an IDE in the `che-editor.yaml` is to specify the `id
 ====
 [source,yaml]
 ----
-id: che-incubator/che-code/insiders
+id: che-incubator/che-idea/latest
```

Comment:
I believe we should keep `che-code/insiders`. It stays interesting for users to switch to the latest Visual Studio Code version.

Reply:
I'm afraid that's the only VS Code version that we offer both upstream and downstream.

Reply:
:) Well, I don't want to confuse readers by showing them how to specify the *default* editor which they explicitly don't need to specify.

Reply:
I believe there is a misunderstanding:

Che (and Dev Spaces) (hopefully) runs by default on a released version of the `che-code` container, such as `che-incubator/che-code/7.56.0`.

The user can switch to the development version of the `che-doc` container: `che-incubator/che-code/insiders`.

See the tag history in quay: https://quay.io/repository/che-incubator/che-code?tab=tags

Therefore, the "Switch to the development version of the Visual Studio Code editor" example still makes sense.

Moreover, the new first class citizen replacing Che-Theia is Visual Studio Code, not IntelliJ, therefore examples should focus on Visual Studio Code as much as possible.

Reply:
After internal discussion:  `che-incubator/che-idea/latest` is the right choice.